pymeasure/instruments/lakeshore/lakeshore_base.py

```diff
@@ -124,3 +121,32 @@ class LakeShoreHeaterChannel(Channel):
         """Control the setpoint temperature
         in the preferred units of the control loop sensor."""
     )
+
+    ramp = Instrument.control(
+        'RAMP? {ch}',
+        'RAMP {ch}, %d, %f',
+        """Control the ramp settings. Needs and replies a tuple (0/1, rate)""",
+    )
```

Comment:
What about a single value: `False` or 0 or something switches it off, while any number switches it on with that value?

That would make the